dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormField.vue

```diff
@@ -0,0 +1,153 @@
+          <FormKit
+            type="text"
+            name="value"
+            v-model="currentValue"
+            :validation-label="validationLabel ?? label"
+            :validation="`number|${validation}`"
+            placeholder="Value"
+            outer-class="short"
+          />
+          <div v-if="unit" class="form-field-label pb-3">
+            <FormKit type="hidden" name="unit" :modelValue="unit" />
+            <span>in {{ unit }}</span>
+          </div>
+          <FormKit
+            v-else-if="options"
+            type="select"
+            name="unit"
+            v-model="currentUnit"
+            placeholder="Currency"
+            :options="options"
+            outer-class="short"
+          />
+        </div>
+      </div>
+      <div>
+        <FormKit type="group" name="dataSource">
+          <div class="next-to-each-other">
+            <div class="flex-1">
+              <UploadFormHeader
+                :label="`${label} Report`"
+                :description="'Select a report as a reference for this data point.'"
+              />
+              <FormKit
+                type="select"
+                name="report"
+                v-model="currentReportValue"
+                placeholder="Select a report"
+                :options="['None...', ...injectReportsName]"
+                :plugins="[selectNothingIfNotExistsFormKitPlugin]"
+              />
+            </div>
+            <div>
+              <UploadFormHeader :label="'Page'" :description="'Page where information was found'" />
+              <FormKit
+                outer-class="w-100"
+                v-model="currentPageValue"
+                type="number"
+                name="page"
+                placeholder="Page"
+                validation-label="Page"
+                step="1"
+                min="0"
+                validation="min:0"
+              />
+            </div>
+          </div>
+        </FormKit>
+      </div>
+
+      <!-- Data quality -->
+      <div class="mb-4" data-test="dataQuality">
+        <UploadFormHeader
+          :label="`${label} Quality`"
+          description="The level of confidence associated to the value."
+          :is-required="isDataQualityRequired"
+        />
+        <div class="md:col-6 col-12 p-0">
+          <FormKit
+            type="select"
+            :modelValue="!isDataQualityRequired ? 'NA' : ''"
+            name="quality"
+            :validation="isDataQualityRequired ? 'required' : ''"
+            validation-label="Data quality"
+            placeholder="Data quality"
+            :options="qualityOptions"
+          />
+        </div>
+      </div>
+      <div class="form-field">
+        <FormKit
+          type="textarea"
+          name="comment"
+          placeholder="(Optional) Add comment that might help Quality Assurance to approve the datapoint. "
+        />
+      </div>
+    </FormKit>
+  </template>
+
+  <template v-else>
```

Comment:
I could not find a case of "DataPointFormField" being mounted without "evidenceDesired = true".
So this v-else seems to actually never happen.
Why is it needed then?

Reply:
But it might be, we wanted to make this datapoint modular and not only usable for cases where a reference is wanted

Reply:
Well I don't think that it is a good idea to implement dead code, because it might be needed in the future.
Let's remove it and add smth like this in the future, when it is actually needed.
Furthermore, we already have other components that work almost the same way as the FormKit in this "v-else" block, so don't really need this behaviour of DataPointFormField here.

Reply:
Removed

Reply:
👍